s3-private/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "project" {}
+variable "environment" {}
+variable "bucket_name" {}
+variable "kms_deletion_window_in_days" {
+  default = 30
+}
+
+# If versioning is enabled then a history of all object changes and deletions is retained.
+variable "versioning" {
+  default = false
```

Comment:
We actually want this on by default for private buckets, ro avoid loss of sensitive data. It's off on public because that data is all compiled and we don't want to keep versions on publically accessible deleted files etc

Reply:
this absolutely makes sense, thank you. https://github.com/dbl-works/terraform/pull/51/commits/89a0065b14494ce505d19e19dc1c827b468ad5f2